toolkit/themes/shared/design-system/design-tokens.json

```
@@ -135,40 +101,42 @@
         "value": "#5a3100"
       }
     },
-    "white": "#ffffff"
-  },
-  "text": {
-    "color": {
-      "prefers-contrast": {
-        "value": "CanvasText"
-      },
-      "deemphasized": {
-        "prefers-contrast": {
-          "value": "inherit"
-        }
-      },
-      "brand": {
-        "light": {
-          "value": "{color.gray.100}"
-        },
-        "dark": {
-          "value": "{color.gray.05}"
-        },
-        "deemphasized": {
-          "light": {
-            "value": "color-mix(in srgb, currentColor 69%, transparent)"
-          },
-          "dark": {
-            "value": "color-mix(in srgb, currentColor 75%, transparent)"
-          }
-        }
+    "white": {
+      "value": "#ffffff"
+    },
+    "background": {
+      "critical": {
+        "value": "{color.red.05}",
+        "darkValue": "{color.red.80}"
+      },
+      "information": {
+        "value": "{color.blue.05}",
+        "darkValue": "{color.blue.80}"
+      },
+      "success": {
+        "value": "{color.green.05}",
+        "darkValue": "{color.yellow.80}"
+      },
+      "warning": {
+        "value": "{color.yellow.05}",
+        "darkValue": "{color.blue.80}"
       }
     }
   },
-  "platform": {
-    "value": "currentColor",
+  "text": {
+    "color": {
+      "value": "CanvasText"
```

```suggestion
      "value": "CanvasText"
```
Not sure what this token should be, maybe I can just copy the format from #3 and this problem will resolve itself. Also not sure what to do about "brand" and "platform" tokens...

---

I think we're just gonna stick with using an empty string + `default` to get around some of the weirdness for now since it makes things work, then re-visit those choices later.